student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+# creating a Person subclass: student
+class Student < Person
+  # Initializes a Student object with the given classroom
+  def initialize(classroom = 'Unknown', name = 'Unknown', age = 0, parent_permission: true)
```

Comment:
- [ ] Kindly refactor the constructor to follow the needed parameters that the Student will receive. `age, classroom, name = "Unknown", parent_permission = true` The only ones that is defined as default are `name` and `parent_permission`